src/Stryker.CLI/README.md

```diff
@@ -54,6 +54,53 @@ All available loglevels are:
 
 `dotnet stryker --max-concurrent-test-runners`
 
+#### Mutation testrun reporters 
+
+`dotnet stryker --reporters ['ConsoleReport', 'ConsoleProgressBar']`
+
+All available reporter options are:
+ * All - This will enable all available reporters
+ * ConsoleProgressBar - This reporter will display the progress of the mutation testrun as a progress bar
+ Example output:
+```
+Tests progress | █████████  | 14 / 15 | 94 % | ~0m 07s |
+Killed : 1
+Survived: 12
+Timeout : 1
+```
+ * ConsoleProgressDots - This reporter will display the progress of the mutation testrun as a dot progress bar in the format .STS...S
```

Comment:
I'm missing an explaination for what each char means.

Reply:
Same :p
I'm guessing there's **S**urvived **T**imeout and **.**Killed. Are there any other possible values?

Reply:
Nope that's all 👍